dev-packages/rollup-utils/plugins/bundlePlugins.mjs

```diff
@@ -121,7 +121,10 @@ export function makeTerserPlugin() {
           // These are used by instrument.ts in utils for identifying HTML elements & events
           '_sentryCaptured',
           '_sentryId',
+          // Keeps the frozen DSC on a Sentry Span
           '_frozenDsc',
+          // This keeps metrics summary on spans
+          '_metrics_summary',
```

Comment:
Ooops turns out metrics summaries never worked for minified CDN bundles 😬 we had no tests covering this, now we have!